pxr/usd/usdMtlx/reader.cpp

```diff
@@ -1458,9 +1460,14 @@ UsdShadeMaterial
 _Context::BeginMaterial(const mx::ConstNodePtr& mtlxMaterial)
 {
     if (TF_VERIFY(!_usdMaterial)) {
+        auto mtlxVersionValue = VtValue(mtlxMaterial->getDocument()->getVersionString());
```

Comment:
I'm guessing this will return one of "" and "1.39" at the moment?  Do we want to handle the fallback case and put 1.38 in instead here?  Or does the document function do that already?